### DISCUSSION TITLE
Allow more actions in toolbar

### DISCUSSION THREAD

**Discussion by wasu-code** [plateaukao/einkbro#413] [Ideas]
Some actions appear in popup triggered from hamburger menu in toolbar, but can't be added to the toolbar itself. 
Is there any reason for this limitation?

One example is "Save EPUB" action. Especially when creting multi-chapter books it would save few clicks to invoke this action right from toolbar.

**Comment by plateaukao** [comment]
just two different implementations.
will try to have more buttons available for toolbar.
